site/src/Navigator.css

```diff
@@ -0,0 +1,198 @@
+div.navigator-box {
+  width: 230px;
+  max-height: 80vh;
+  background: rgba(255, 255, 255, 0.9);
+  padding: 13px 10px;
+  border-radius: 10px;
+  margin: 3px;
+  box-shadow: 0 2px 4px white;
+  border-color: rgba(15, 15, 15, 0.8);
+  border: 3px;
+  overflow: hidden;
+}
+
+div.navigator-container {
+  overflow-y: auto;
+  width: 210px;
+  height: 416px;
+  scrollbar-color: var(--ifm-color-secondary-dark) rgba(0, 0, 0, 0);
```

Comment:
oooh this is for my own information but- where's this .css variable coming from? Looks like infinima?